google/datalab/bigquery/_table.py

```diff
@@ -103,8 +103,11 @@ class Table(object):
   # Allowed characters in a BigQuery table column name
   _VALID_COLUMN_NAME_CHARACTERS = '_abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789'
 
-  # When fetching table contents, the max number of rows to fetch per HTTP request
+  # When fetching table contents for a range or iteration, use a small page size per request
```

Comment:
Sure, here are some numbers: :)

| Page size | Fetch time |
| ------------- | ------------- |
| 50 | ~160ms |
| 100 | ~180ms |
| 1024 | ~700ms |
| 10,000 | ~5s |

So we need to minimize latency, because going the 100k route will have the users waiting for ~5 seconds to get their first page (even though they probably won't wait for another fetch). This also applies if you want to use the table iterator to get rows 1000 to 1020 for example.